### DISCUSSION TITLE
flatpak update error with MESA package in eOS 7

### DISCUSSION THREAD

**Discussion by Kazekage-Gaara** [elementary/appcenter#1985] [Q&A]
Hello, since yesterday I installed ElementaryOS 7 and I performed an update just installed the system. But I always have a pending update in the application center apparently a flatpak package. When I try from the terminal running the flatpak update command, I get the following error:

Error: Error pulling from repo: While loading runtime/org.freedesktop.Platform.GL.default/x86_64/22. 08 from the remote freedesktop repository: fsck 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d.dirtree: No such metadata object 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d.dirtree

When I run the flatpak repair command it returns;
error: No such metadata object 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d.dirtree

Apparently it's a bug with MESA, I don't know what to do anymore.

**Comment by lenemter** [comment]
You probably ran `flatpak repair` without `sudo` so please try to execute `sudo flatpak repair` and see if it resolves the issue.

**Reply by Kazekage-Gaara** [reply]
hello, I am running the command as sudo.
 
[sudo] contraseña para kazekage:        
[32/32] Verificando appcenter:runtime/org.gnome.Epiphany.Locale/x86_64/stable…
Comprobando remotos...
Podando objetos
error: No such metadata object 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d.dirtree

**Reply by lenemter** [reply]
Hm... Quick googling suggests running `flatpak repair --user` too. If that does not work try to reinstall flatpak platform package using `flatpak install --reinstall org.freedesktop.Platform.GL.default/x86_64/22.08`. I think this should resolve the issue.

**Reply by Kazekage-Gaara** [reply]
flatpak install --reinstall org.freedesktop.Platform.GL.default/x86_64/22.08
Buscando coincidencias…


        ID                                             Rama             Op            Remoto                Descarga
 1. [✗] org.freedesktop.Platform.GL.default            22.08            i             freedesktop           1,0 kB / 137,7 MB

Error: When attempting to review cd01feb2d3ea1418af823a631d4df8469b5c878bb69cdd6f041b90bade0c10f4 in /var/lib/flatpak/runtime/org.freedesktop.Platform.GL.default/x86_64/22.08/.cd01feb2d3ea1418af823a631d4df8469b5c878bb69cdd6f041b90bade0c10f4-XRUAZ1: No such metadata object 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d.dirtree
error: Failed to instalar org.freedesktop.Platform.GL.default: When attempting to review cd01feb2d3ea1418af823a631d4df8469b5c878bb69cdd6f041b90bade0c10f4 in /var/lib/flatpak/runtime/org.freedesktop.Platform.GL.default/x86_64/22.08/.cd01feb2d3ea1418af823a631d4df8469b5c878bb69cdd6f041b90bade0c10f4-XRUAZ1: No such metadata object 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d.dirtree

**Reply by lenemter** [reply]
I have no idea why this is happening. Have you tried running `flatpak repair --user` too? Also some people suggest running `flatpak uninstall --unused`. I can't suggest anything other than that unfortunately :(
